middleware/static.go

```diff
@@ -157,9 +156,9 @@ func StaticWithConfig(config StaticConfig) echo.MiddlewareFunc {
 	}
 
 	// Index template
-	t, err := template.New("index").Parse(html)
-	if err != nil {
-		panic(fmt.Sprintf("echo: %v", err))
+	t, tErr := template.New("index").Parse(html)
```

Comment:
Better keep using `err`

Reply:
I wanted to make sure that we would not happen to have something similar like https://github.com/labstack/echo-jwt/pull/4 had. Err instance is created when we create the middleware function and that function is modified to reuse that same instance - but functions are now run in request goroutine. Meaning we have data race.

this is little bit paranoid approach

Reply:
Agreed